dist-git.spec

```diff
@@ -90,65 +89,58 @@ getent group gen-acls > /dev/null || \
 
 getent passwd gen-acls > /dev/null || \
     useradd -r -g gen-acls -G packager -s /bin/bash \
-            -d %{_sharedstatedir}/dist-git/git gen-acls
+            -d /srv/git gen-acls
 
 
 %install
 # ------------------------------------------------------------------------------
-# /usr/share/ .... static files
+# /usr/local/bin ........... scripts
 # ------------------------------------------------------------------------------
-install -d %{buildroot}%{_datadir}/dist-git
-
-cp -a scripts/dist-git/* %{buildroot}%{_datadir}/dist-git/
+install -d %{buildroot}/usr/local/bin/
+cp -a scripts/dist-git/* %{buildroot}/usr/local/bin/
 
 # ------------------------------------------------------------------------------
 # /etc/ .......... config files
 # ------------------------------------------------------------------------------
 install -d %{buildroot}%{_sysconfdir}/dist-git
+install -d %{buildroot}%{_sysconfdir}/gitolite
 install -d %{buildroot}%{_sysconfdir}/httpd/conf.d/dist-git
-install -d %{buildroot}%{_sysconfdir}/cron.d/dist-git
 mkdir -p   %{buildroot}%{_unitdir}
 
 cp -a configs/dist-git/dist-git.conf  %{buildroot}%{_sysconfdir}/dist-git/
-cp -a configs/gitolite/gitolite.rc    %{buildroot}%{_sysconfdir}/dist-git/
 cp -a configs/httpd/dist-git.conf     %{buildroot}%{_sysconfdir}/httpd/conf.d/
+cp -a configs/gitolite/gitolite.rc    %{buildroot}%{_sysconfdir}/gitolite/
 cp -a configs/httpd/ssl.conf.example  %{buildroot}%{_sysconfdir}/httpd/conf.d/
 cp -a configs/httpd/dist-git/* %{buildroot}%{_sysconfdir}/httpd/conf.d/dist-git/
-cp -a configs/cron/*           %{buildroot}%{_sysconfdir}/cron.d/dist-git/
 cp -a configs/systemd/*        %{buildroot}%{_unitdir}/
 
+install -d %{buildroot}%{_sysconfdir}/gitolite
+install -d %{buildroot}%{_sysconfdir}/gitolite/conf
+install -d %{buildroot}%{_sysconfdir}/gitolite/logs
+install -d %{buildroot}%{_sysconfdir}/gitolite/local
+install -d %{buildroot}%{_sysconfdir}/gitolite/local/VREF
+install -d %{buildroot}%{_sysconfdir}/gitolite
+install -d %{buildroot}%{_sysconfdir}/gitolite/hooks
+install -d %{buildroot}%{_sysconfdir}/gitolite/hooks/common
+
```

Comment:
-   install dist-git and copr-dist-git packages
-   mount the persistent volumes
-   start the service

You need to mount to `/srv/cache/lookaside/pkgs` directly for it. Similarly for Git repositories, that would be `/srv/git/repositories`.